.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Feature Request
+description: Request a feature that you would like Eludris to support
```

Comment:
like the previous one, "Please enter the details of the feature you want added" or something along the lines